docs/score_tools/rustfmt.rst

```diff
@@ -0,0 +1,138 @@
+
+.. doc_tool:: rustfmt
+   :id: doc_tool__rustfmt
+   :status: evaluated
+   :version: >= 1.8.0
+   :tcl: HIGH
+   :safety_affected: YES
+   :security_affected: YES
+   :realizes: wp__tool_verification_report
+   :tags: tool_management
+
+Rustfmt Verification Report
+===========================
+
+Introduction
+------------
+Scope and purpose
+~~~~~~~~~~~~~~~~~
+Rustfmt is the standard formatting tool for the Rust programming language. It automatically
+formats Rust code according to style guidelines defined in the Rust community. By using Rustfmt,
+developers can ensure that their code adheres to a consistent style, improving readability and
+maintainability across projects. Rustfmt can be integrated into development workflows, including
+IDEs and continuous integration pipelines, to enforce coding standards automatically.
+
+Inputs and outputs
+~~~~~~~~~~~~~~~~~~
+| Inputs: Software sources (Rust)
+| Outputs: Formatted software sources (Rust)
+
+.. figure:: _assets/rustfmt.drawio.svg
+  :width: 100%
+  :align: center
+  :alt: Rustfmt overview
+
+  Rustfmt overview
+
+Available information
+~~~~~~~~~~~~~~~~~~~~~
+- Version: >= 1.8.0
+- Official repository: https://github.com/rust-lang/rustfmt
+- Official documentation: https://rust-lang.github.io/rustfmt
+- Rustfmt configuration in S-CORE module repository: https://github.com/eclipse-score/score/issues/2011
+
+
+Installation and integration
+----------------------------
+Installation
+~~~~~~~~~~~~
+| To add the Rustfmt Bazel targets to your project or module, include the following line in your MODULE.bazel file:
+
+.. code-block:: Python
+
+  To be added once https://github.com/eclipse-score/score/issues/2011 is done
+
+
+Integration
+~~~~~~~~~~~
+Integrated in bazel.
+
+Environment
+~~~~~~~~~~~
+Requires Rust toolchain and Bazel build environment.
+
+Safety evaluation
+-----------------
+This section outlines the safety evaluation of Rustfmt for its use within the S-CORE project.
+
+
+.. list-table:: Safety evaluation
+   :header-rows: 1
+   :widths: 1 2 8 2 6 4 2 2
+
+   * - Malfunction identification
+     - Use case description
+     - Malfunctions
+     - Impact on safety?
+     - Impact safety measures available?
+     - Impact safety detection sufficient?
+     - Further additional safety measure required?
+     - Confidence (automatic calculation)
+   * - 1
+     - Run formatting on source files and does not format according to rules
+     - | The source code is not formatted according to the defined style guidelines, leading to inconsistent code style across the project.
+     - no
+     - no
+     - yes
+     - no
+     - high
+   * - 2
+     - Run formatting on source files changes code unintentionally
+     - | The source code changes its logical structure or behavior due to incorrect formatting, potentially introducing bugs.
+     - yes
+     - | Likelihood: Low. Rustfmt is used in virtually every Rust project. This gives high confidence in its quality
```

Comment:
As far as I have experience in tool evaluations outside S-CORE, such comparisons to other projects only work when the tool version and execution environment are identical.